csharp/GSDK_CSharp_Standard/Properties/PublishProfiles/FolderProfile.pubxml

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
+https://go.microsoft.com/fwlink/?LinkID=208121.
+-->
+<Project>
+  <PropertyGroup>
+    <Configuration>Release</Configuration>
+    <Platform>Any CPU</Platform>
+    <PublishDir>C:\Users\digkanat\Desktop\lala</PublishDir>
```

Comment:
Why is it set to your personal directory?

Reply:
nice catch, removed!